data/sql/updates/pending_db_world/rev_1646669102474392500.sql

```diff
@@ -0,0 +1,200 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1646669102474392500');
+
+DELETE FROM `quest_greeting` WHERE `ID` BETWEEN 234 AND 186420;
```

Comment:
this makes me a bit nervous as that is a pretty huge range between. are we certain you got everything between this range in the insert below?

Reply:
Yes, the table is currently empty. It was just introduced by https://github.com/azerothcore/azerothcore-wotlk/pull/10526 this past weekend. However, if we would prefer I can have it only remove the entries explicitly added. Should I make that change?

Reply:
I have updated PR to only remove the IDs only specifically added beneath. Thank you for the suggestion! I think it is a good change.